moving-forward-series/simple-payment-verification.md

```diff
@@ -0,0 +1,50 @@
+## Can a Crypto Wallet be Lightweight, Private, AND Secure? The SPV Model Leads the Way.
+
+People need mobile wallets that are fast to set up, easy to use and very secure. All too often, mobile wallets make extreme security sacrifices in order to slightly improve ease of use. Including, the provision for a centralised node to serve requests to the wallet software. This is a trust model that processes all wallet information, sent and received, through a single point of failure. SPV challenges this convention by providing a high level of security whilst improving the user experience. 
```

Comment:
* This is not a complete sentence: "Including, the provision for a centralized node..." 
    * Change the sentences to this:
        * One example of a bad security practice is when a wallet has to trust a centralised node.
* This first paragraph is the very first time you mention SPV (other than the title, which is perfectly fine to leave as-is.) Therefore, you should spell out SPV the first time you use it in your article.
    * Change to:
        * A wallet using a Simplified Payment Verification (SPV) model challenges this convention by providing...